core/src/proc_macros_support.rs

```diff
@@ -0,0 +1,23 @@
+use jsonrpsee_types::ErrorObjectOwned;
+
+#[cold]
```

Comment:
> cold marks all the basic blocks that dominate the call as cold, and thus any branches that lead to them as automatically-unlikely. It then uses that to structure the generated assembly so that the likely blocks are proximate -- if you've ever noticed that call paths which end up panicking are always at the end of the assembly for a function, out of the way of the normal paths, this is why.